tests/other/noffi-side.cpp

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2019 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <cstdint>
+extern "C"{
+    extern void passBigInt(uint64_t);
+}
+
+void callPassBigInt(){
+    passBigInt(1152921504606846975);
```

Comment:
I think we expect that this would get translated into a call_indirect, but this gets translated into a  $env._passBigInt instead.

Reply:
The indirection here is handled on the JS side.  The imported function is a JS stub that gets filled in as symbols are resolved by loading modules.

Reply:
right and this JS stub is causing problems when we turn off JS legalization.